src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1736,12 +1736,30 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode)
                             ->useAttributeAsKey('name')
                             ->arrayPrototype()
                                 ->children()
-                                    ->scalarNode('lock')->defaultValue('lock.factory')->end()
-                                    ->scalarNode('storage')->defaultValue('cache.app')->end()
-                                    ->scalarNode('strategy')->isRequired()->end()
-                                    ->integerNode('limit')->isRequired()->end()
-                                    ->scalarNode('interval')->end()
+                                    ->scalarNode('lock_factory')
+                                        ->info('The service ID of the lock factory used by this limiter')
+                                        ->defaultValue('lock.factory')
+                                    ->end()
+                                    ->scalarNode('cache_pool')
+                                        ->info('The cache pool to use for storing the current limiter state')
+                                        ->defaultValue('cache.app')
+                                    ->end()
+                                    ->scalarNode('storage_service')
+                                        ->info('The service ID of a custom storage implementation, this precedes any configured "cache_pool"')
+                                        ->defaultNull()
+                                    ->end()
+                                    ->enumNode('strategy')
+                                        ->info('The rate limiting algorithm to use for this rate')
+                                        ->isRequired()
+                                        ->values(['fixed_window', 'token_bucket'])
+                                    ->end()
+                                    ->integerNode('limit')
+                                        ->info('The maximum allowed hits in a fixed interval or burst')
+                                        ->isRequired()
+                                    ->end()
+                                    ->scalarNode('interval')->info('Configures the fixed interval if "strategy" is set to "fixed_window"')->end()
                                     ->arrayNode('rate')
+                                        ->info('Configures the fill rate if "strategy" is set to "token_bucket"')
```

Comment:
Same here: `Configures the fill rate` + `in percent, from 0.0 to 1.0` or something like that?